src/collective/volto/formsupport/restapi/services/submit_form/post.py

```diff
@@ -413,6 +413,7 @@ def prepare_message(self):
             "parameters": self.filter_parameters(),
             "url": self.context.absolute_url(),
             "title": self.context.Title(),
+            "block": self.block,
```

Comment:
do not pass the entire block, but only the pieces that you need.

Reply:
no

Reply:
u also have view and portal in the view if i remember, if u pass the block is not changing the performance and keeps the logics so as the view is about the compiled block.